community/governance/sigs.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: single
+title:  "Special Interest Groups (SIGs)"
+sidebar:
+  nav: community
+---
+
+SIGs exist so members of the community can discuss some of Spinnaker's core
+feature areas, and guide design in those areas.
+
+* SIGs are created as needed
+* Each SIG has one or more "leads," who own the SIG's agenda
+* Anyone can attend a SIG meeting
+
+# Existing SIGs
+
+Below is a list of the current Spinnaker SIGs.
+
+| Name | Lead(s) | Agenda/Notes | Meeting Link | Contact |
+| - | - | - | - | - |
+| __Kubernetes__ | lwander@ | [Link](https://docs.google.com/document/d/1db_yw1uru99Byvin4lQgm7aUZ9xMmvsARtEiiNUrmBo) | [Link (Hangouts)](https://plus.google.com/hangouts/_/google.com/kubernetes-v2?hceid=bHdhbmRlckBnb29nbGUuY29t.64nbl442ak31mn3cl3b667uevf&authuser=0) | [spinnaker-sig-kubernetes@googlegroups.com](https://groups.google.com/forum/#!forum/spinnaker-sig-kubernetes), [#sig-kubernetes on Slack](https://spinnakerteam.slack.com) |
```

Comment:
Any good way to list/indicate next meeting date/time(s)? I see it indicated on the agenda, and we of course don't want 2 places to keep this info. More than anything, it should be specific and accurate.

Reply:
I don't see an easy way to list them without updating this page every week with the next meeting, since it's biweekly.

Reply:
Sgtm